crates/iota-framework/packages/iota-system/sources/validator_set.move

```diff
@@ -1129,4 +1490,153 @@ module iota_system::validator_set {
         };
         res
     }
+
+    public(package) fun committee_validator_addresses(self: &ValidatorSetV2): vector<address> {
+        let vs = &self.active_validators;
+        let num_validators = vs.length();
+        let committee_members = &self.committee_members;
+
+        let mut res = vector[];
+        let mut i = 0;
+        let committee_members_num = committee_members.length();
+        while (i < committee_members_num) {
+            let validator_index = committee_members[i];
+            assert!(
+                validator_index < num_validators,
+                ECommitteeMembersSetCorrupt,
+            );
+            let validator_address = vs[validator_index].iota_address();
+            res.push_back(validator_address);
+            i = i + 1;
+        };
+        res
+    }
+
+    // Selects new set committee members from the set of all active validators.
+    // Currently selects all active validators to be part of the committee.
+    public(package) fun select_committee_members(self: &mut ValidatorSetV2) {
```

Comment:
Is this even used somewhere? Don't we want to directly use `top_n_stakers`?

Reply:
Right, removed it.